spec/cli_spec.rb

```diff
@@ -3,10 +3,23 @@
 require 'spec_helper'
 
 APP_DIR = File.expand_path('app', __dir__)
+AFTER_HOOK_APP_DIR = File.expand_path('after_hook', __dir__)
 
 RSpec.describe 'CLI' do
   it 'renders reproducible dumps' do
     system('bundle', 'exec', 'rails-response-dumper', chdir: APP_DIR, exception: true)
     expect(File.join(APP_DIR, 'dumps')).to match_snapshots
   end
+
+  it 'runs after hooks when an exception is raised' do
+    system(
+      'bundle',
+      'exec',
+      'rails-response-dumper',
+      chdir: AFTER_HOOK_APP_DIR,
+      exception: false,
+      err: File::NULL
```

Comment:
Could we assert the contents of stderr? It should contain the raised exception or other useful information.